src/null-safety/migration-guide.md

```diff
@@ -63,7 +63,7 @@ The instructions assume your code is under **source control**,
 so that you can easily undo any changes.
 
 
-### Switch to a 2.12 beta release
+### Switch to the latest 2.12 beta release
```

Comment:
We need to call out Flutter explicitly, right?

We need to make sure users use *at least* the latest beta, but *NOT* dev or master. Or they risk messing things up!

Reply:
I'm not sure I follow; the text below already 1) says to pick Dart or Flutter, and 2) to use beta ?

Reply:
Could we just say "Switch to the latest beta release"?